templates/emby/0/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+emby:
+  image: emby/embyserver:latest
+  net: host
```

Comment:
It was specified by the old documentation. The new way is a bit different and I'm not sure what it implicates talking about networking.

With the old system still works, but without the `net: host` probably won't. Emby needs a lot of ports due to the amount of services it has.